tests/sdk/python/helm-local/step1/requirements.txt

```diff
@@ -1 +1 @@
-pulumi>=2.0.0,<3.0.0
+pulumi>=3.0.0,<4.0.0
```

Comment:
Does this need to be >= 3.9.0 as well?

Reply:
Hmm, must be getting overridden somewhere, but I'll update.